src/plugins/i18n.ts

```diff
@@ -121,11 +121,47 @@ const messages = {
       },
     },
     register: {
-      oauth: 'Or register with',
+      title: 'Register to Reden',
+      invalid: {
+        email: 'Invalid email address.',
+        password: {
+          strength:
+            'Password is not strong enough, please use at least 8 characters, including uppercase, lowercase and numbers.',
+          mismatching: 'Passwords do not match',
+        },
+        username: 'Invalid Username.',
+      },
+      existing: 'Already have an account?',
+      login: 'Login',
+      sent: {
+        title: 'One more step',
+        message:
+          "We have sent an email to {email} with a link to complete your registration.If you don't see the email, please check other places it might be, like your junk, spam, social, or other folders.",
```

Comment:
I mean this, you're violating **English** grammar. Add a space after the periods and commas.